test/test_distributions_mixture.py

```diff
@@ -78,6 +85,46 @@ def test_mixture_broadcast_batch_shape(
     _test_mixture(mixing_distribution, component_distribution)
 
 
+@pytest.mark.parametrize("batch_shape", [(), (1,), (7,), (2, 5)])
+@pytest.mark.parametrize("validate_args_for_normal", [False, True])
+@pytest.mark.parametrize("validate_args_for_half_normal", [False, True])
```

Comment:
Setting `validate_args=True` in the components seems unnecessary to me (assume that `support` is provided like below).